### PR TITLE
Fix typos and update several links in notebook «Advanced RAG on Hugging Face»

### PR DESCRIPTION
**Pull request by rnckp**
- Fix some typos
- Add some missing words
- Update several links

Btw: Very helpful notebooks - thanks for making this available. Very much appreciated. 👍

### CODE REVIEW

File: notebooks/en/advanced_rag.ipynb [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "id": "hUCaGdAj9-9F"
```
Comment (aymeric-roucher) [inline notebooks/en/advanced_rag.ipynb:6]: <p>The addition of <code>.ipynb</code> will break the link, which for now works (test it <a href="https://huggingface.co/learn/cookbook/advanced_rag" target="_blank">here</a>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by aymeric-roucher** [comment]
After undoing the change that I mentioned, all will be perfect! Thank you for adding these changes @rnckp 🤗

File: notebooks/en/advanced_rag.ipynb (hunk 1 above)
Comment (rnckp) [reply notebooks/en/advanced_rag.ipynb:6]: Ah, interesting... That makes sense. I tried the notebook in VS Code. There the link w/o `.ipynb` suffix didn't work. So VSCode seems to interpret these links differently. 👍

Comment (rnckp) [reply notebooks/en/advanced_rag.ipynb:6]: Done!

Comment (rnckp) [reply notebooks/en/advanced_rag.ipynb:6]: @aymeric-roucher Can I do more to help with the merge? I have pushed the change as you suggested. 
Thanks and have a great day!

Comment (merveenoyan) [inline notebooks/en/advanced_rag.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #18.        chunk_size=1000,  # The maximum number of characters in a chunk: we selected this value arbitrarily.
</pre><p>Let's not add any dots at the end for consistency across the cookbook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (merveenoyan) [inline notebooks/en/advanced_rag.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #13.    # Plot the distrubution of document lengths, counted as the number of tokens.
</pre><p>Typo here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Comment by merveenoyan** [comment]
@rnckp thanks for the nits, I left two comments

File: notebooks/en/advanced_rag.ipynb (hunk 1 above)
Comment (rnckp) [reply notebooks/en/advanced_rag.ipynb:6]: Thanks. I have removed all dots from comments.

Comment (rnckp) [reply notebooks/en/advanced_rag.ipynb:6]: I have corrected this typo too.

**Comment by rnckp** [comment]
@merveenoyan @aymeric-roucher I have changed the code according to your comments. 👍

**Review by merveenoyan — APPROVED** [review]
thank you for working on this!

**Pull request merged**